src/api/serializers.py

```diff
@@ -67,13 +68,27 @@ class Meta:
         many=True
     )
 
+class PreprintSubjectSerializer(serializers.HyperlinkedModelSerializer):
 
+    class Meta:
+        model = repository_models.Subject
+        fields = ('name',)
+
+class PreprintFileSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = repository_models.PreprintFile
+        fields = ('original_filename', 'mime_type', 'download_url',)
+
+class PreprintSupplementaryFileSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = repository_models.PreprintSupplementaryFile
+        fields = ('url', 'label',)
 class IssueSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = journal_models.Issue
-        fields = ('pk', 'volume', 'issue', 'issue_title', 'date', 'issue_type', 'issue_description',
```

Comment:
Can you tell us why you removed this whole line?

Reply:
I'm pretty sure you asked me to 'remove this, please', in an earlier line comment

Reply:
I think I asked you to undo the change you made on that line...